afterpay/src/main/kotlin/com/afterpay/android/Afterpay.kt

```diff
@@ -27,6 +28,10 @@ object Afterpay {
     internal val locale: Locale
         get() = configuration?.locale ?: Locales.US
 
+    // TODO: remove this
```

Comment:
Why the TODO?

Reply:
Thanks for the pickup, thought i'd removed it before pushing. Removed in 368e21f